Desktop version

Sirvelasque opened the pull request:
- re-rize width and height properties to make use of the desktop screen
- add input for last name and signal/button to go to work section both doesn't display on mobile version
- adjusted work section structure by changing the flex direction so it stick to the desktop template
- adjusted width and height on work's picture spaces making them responsive

Sirvelasque committed:
add my email and media icons position

correction on media icons padding and added my personal email in the footer

Sirvelasque committed:
add second work button for desktop

add second work button which will scroll down to the work section and will only display on the desktop version, change the container direction to row so they display in both sides

Sirvelasque committed:
re-size width on work section

change the parameter of the inside container to make them responsive to then change the values on desktop version

Sirvelasque committed:
made work section desktop version

change the flex-direction and made adjustments on the width and height, also changed the width value to get into mobile mode to 930px

Sirvelasque committed:
adjust top elipses for desktop version

made a copy of the ellipses and switch the display from none to flex on respective width

Sirvelasque committed:
finish adjust on elipses for desktop version

adjust width and height for the most of the ellipses and all their top and left properties

Sirvelasque committed:
fix footer and contact formfor desktop version

display new input for last name in desktop version and fixes the properties so it doesn't move weird on shrink

Sirvelasque committed:
set headline properties for desktop version

move support text, dividers and change the width so it take more of the desktop width

Sirvelasque committed:
fix linter errors

na

Sirvelasque committed:
linter errors fixing

Sirvelasque committed:
fixing linter errors

Sirvelasque committed:
linter error fixed

t-yanick commented on css/style.css:
```diff
@@ -465,3 +556,151 @@ footer {
+  color: #ebebff;
+}
+
+@media screen and (min-width: 930px) {
```
- [ ] Since it is a requirement for this project, your desktop breakpoint value for `min-width` should be in the range `992px - 1100px` Please kindly fix that. See screenshot of the requirements below

![range](https://user-images.githubusercontent.com/62849872/150656890-9b48c7e3-9b5f-421a-84d3-32c1372d00b9.png)

t-yanick commented on html/index.html:
```diff
@@ -78,59 +82,70 @@ <h2>Software developer</h2>
+              <li class="tool_tag">bootstrap</li>
+              <li class="tool_tag">Ruby</li>
+            </ul>
+              <button type="button" class="project_btn">See project</button>
```
- As it is a requirement for this project and required skill for a front-end dev, sticking to the design as much as possible, please kindly fix the implementation of the `See project` buttons to match the design. The text is overlapping, see screenshots below and kindly fix that. 

***The design***
![theD](https://user-images.githubusercontent.com/62849872/150657080-bc0191a5-cf02-4562-9d0c-9f8a1c4ddd32.png)

***Your Implementation***
![yourI](https://user-images.githubusercontent.com/62849872/150657093-0deb9c3c-e2ca-4433-8296-34432f0e48c3.png)

medaminedev66 replied:
[OPTIONAL]
- Kindly remove the border from the `see project` button to make your design better.

t-yanick reviewed (changes requested):
Hi @Sirvelasque, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments below and under the review._ 
### I will leave the comment here, since your `Readme file` is not accessible for this PR ###

- [ ] Please kindly update your `readme file` to reflect the changes for this milestone as this will make your `readme file` more informative and professional. Checkout this [template](https://github.com/microverseinc/readme-template) and adapt it to your project. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Sirvelasque committed:
fix button's bug and change responsive condition

re-size padding and width to avoid the letter on the button to change the order in some devices and fixed the condition to switch between desktop and mobile version

Sirvelasque committed:
update readme file

Sirvelasque committed:
change formulari's list class

Sirvelasque committed:
Merge branch 'desktop_version' of https://github.com/Sirvelasque/portfolio into desktop_version

Sirvelasque committed:
empty line remove

Sirvelasque committed:
fix space between form items

Sirvelasque committed:
fix linter errors

Sirvelasque committed:
linter error fix try

Sirvelasque committed:
Update style.css

medaminedev66 commented on css/style.css:
```diff
@@ -465,3 +557,163 @@ footer {
+  color: #ebebff;
+}
+
+@media screen and (min-width: 992px) and (max-width: 1100px) {
```
- [ ] [REQUIRED] Your website is still on the mobile version. Kindly remove "` and (max-width: 11000px)`" so your desktop version design will be implemented.
- Your website:
<img width="698" alt="image" src="https://user-images.githubusercontent.com/77190203/150659875-0696a1cb-82ec-44b6-b613-41812a02d4f9.png">

Sirvelasque replied:
@medaminedev66 sorry but I'm confused, the last reviewer told me to implement it, and it works properly on my computer in the gap btween the sizes

medaminedev66 replied:
The previous reviewer said that the `min-width` should be between 992px and 1100px. He didn't mention anything about `max-width`. Kindly read the requested changes carefully. By adding `max-width: 1100px` the desktop version will not be working on screens that are larger than 1100px.

medaminedev66 commented on html/index.html:
```diff
@@ -78,59 +82,70 @@ <h2>Software developer</h2>
         <!--divider-->
-        <img alt="divider vector" src="../img/Vector.svg" class="divider_c">
+        <div class="divider_c"></div>
       </div>
+
       <div class="works_container">
```
[OPTIONAL]
- You implemented the desktop version in a good way. Let's make it better by reducing the margin between sections.
What do you think? I will also  recommend reducing the margin between the projects.
<img width="701" alt="image" src="https://user-images.githubusercontent.com/77190203/150660074-ffe02f68-729c-4510-9231-f2d29f9a3c63.png">

medaminedev66 reviewed (changes requested):
Hi @Sirvelasque , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly remove the gap that is on the right of your portfolio to improve your UI design:
<img width="570" alt="image" src="https://user-images.githubusercontent.com/77190203/150660294-4090cb71-77a9-4089-a973-acd1116fa9a5.png">


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Sirvelasque committed:
remove border from button change mobile version max width

Sirvelasque committed:
adjust work seccion width, heigh and margin between projects

Sirvelasque committed:
fix linter errors

mateo951 reviewed (approved):
### Hi @Sirvelasque, 

Good work on addressing all the requested changes asked by the previous reviewers 🔥🔥🔥
Your project is complete! There is nothing else to say other than... it's time to merge it 👌😎 
#### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
